pkg/reading/names.go

```diff
@@ -0,0 +1,29 @@
+package reading
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Names struct {
+	CustomFields map[string]interface{}
+}
+
+func (n *Names) UnmarshalJSON(data []byte) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method Names.UnmarshalJSON should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)